### ISSUE TITLE
KDE Login Screen - Type Clock unavailable &  qt-quickcontrols 2.5 not installed

### ISSUE THREAD

**Issue by hellowtisch** [vinceliuice/McMojave-kde#14]
Hi, I do get the above errors, and am stuck here... I do have a fresh minimal kubuntu installation.

Are there any packages, I should install? See the list of installed packages attached.
[packages.txt](https://github.com/vinceliuice/McMojave-kde/files/3908069/packages.txt)

Thanks for a hint.

**Comment by syhgorath** [comment]
i have the same problem on kubuntu 18.04

**Comment by aditya12398** [comment]
Can we get a response on this issue please? I am facing the same and I cant seem to have found a solution anywhere.

**Comment by aditya12398** [comment]
So I found a workaround (or a solution? I am not sure.)
You have to change the version for QtQuick.Controls in the Clock.qml file to 1.1 (same as the one in Main.qml)
I did the following and that made it work.

```sudo nano /usr/share/sddm/themes/McMojave/components/Clock.qml```

change the line `import QtQuick.Controls 2.5` to `import QtQuick.Controls 1.1`

Save and exit. Logout and login again, see if the theme is working now.

**Comment by JuenTingShie** [comment]
Thanks it works!

**Comment by Hawkbawk** [comment]
I've been having this issue on a fresh install of Kubuntu 18.04, with all packages up to date. As far as I can tell, this is because the default version of Qt that Kubuntu is setup to use is 5.9.5, which doesn't have QtQuick.Controls >= 2.5, which would explain our issues.

The fix suggested by [aditya12398](https://github.com/vinceliuice/McMojave-kde/issues/14#issuecomment-583252625) seems to be the best fix out there. The only other options are basically to ibuild Qt 5.12 from source and hope that fixed things, or try and add one of these third-party [PPA's](https://launchpad.net/~beineri)and edit some default configuration files.

Apparently though, this is a somewhat hacky fix at best, due to some [path and/or import issues](https://unix.stackexchange.com/questions/481063/ubuntu-18-04-1-lts-x64-how-to-update-qt-5-10-from-qt-5-9-5).

Basically, until the default version of Qt shipped with Kubuntu is updated to >= 5.11, we're stuck editing config files. Until then be patient. Who knows, maybe it'll show up in Kubuntu 20.04?

**Comment by mskramst** [comment]
Yes, Changing to 1.1 does the trick for a couple I tried.
